routes/api.js

```diff
@@ -70,6 +81,47 @@ router
     }
 
     res.json({ authenticated: !!user });
+  })
+  .post('/no_shows', isUserAuthenticated, async (req, res, next) => {
+    try {
+      const { access_token, refresh_token } = req.user;
+      const { invitee } = req.body;
+
+      const calendlyService = new CalendlyService(access_token, refresh_token);
+
+      const { resource } = await calendlyService.markAsNoShow(invitee);
+
+      res.json({ resource });
+    } catch (error) {
+      next(error);
+    }
+  })
+  .delete('/no_shows/:uuid', isUserAuthenticated, async (req, res, next) => {
+    try {
+      const { access_token, refresh_token } = req.user;
+      const { uuid } = req.params;
+      const calendlyService = new CalendlyService(access_token, refresh_token);
+
+      await calendlyService.undoNoShow(uuid);
+
+      res.status(204).send('No-show successfully undone');
+    } catch (error) {
+      next(error);
+    }
+  })
+  .get('/no_shows/:uuid', isUserAuthenticated, async (req, res, next) => {
```

Comment:
I don't _think_ we're using this endpoint?

Reply:
Correct. I had this to only allow no-show undoing if the invitee is, indeed, a no-show. I've worked this out via the UI where the undo button only shows if invitee is a no-show. Deleting! Also deleting the "findNoShow" method.